ner_v1/language_utilities/utils.py

```
@@ -1,31 +1,87 @@
+
+
+def make_request(url):
+    """
+    Method to make a request call and return data for given url
+    Args:
+        url (str): url to which to make request
+    Returns:
+        (str): returns url response data
+    """
+    return urllib.urlopen(url).read()
+
+
+def get_translate(text, target, source):
+    """
+    Method to get raw response data of google translation API to translate given text from target
+    language to source language
     Args:
-        text: Text snippet which needs to be translated
-        source_language_code: ISO-639-1 code for language script corresponding to text ''
-        target_language_code: ISO-639-1 code for target language script
-    Return: 
-        dict: Dictionary containing two keys corresponding to 'status'(bool) and 'translated text'(unicode)
+        text (str): text to be translated
+        target (str): language code in which text will be translated
+        source (str): language code of the given text
+    Returns:
+        (dict): dict containing response
         For example: Consider following example
-                     text: 'नमस्ते आप कैसे हैं'
-                     'source_language_code': 'hi'
-                     'target_language_code': 'en'
-                     
-                     translate_text(text, 'hi', 'en')
-                     >> {'status': True,
-                        'translated_text': 'Hello how are you'}
+                 text: 'नमस्ते आप कैसे हैं'
+                'source': 'hi'
+                'target': 'en'
+
+                >> get_translate(text, 'hi', 'en')
+                >> {u'data': {u'translations': [{u'translatedText': u'\u092e\u0947\u0930\u093e \u0928\u093e\'}]}}
+    """
+    query_params = {"q": text, "source": source, "target": target}
+    url = TRANSLATE_URL + "&" + unicode_urlencode(query_params)
+    try:
+        return demjson.decode(make_request(url))
+    except Exception, e:
```

Better let this fail instead of catching